### ISSUE TITLE
RocketChat bridge doesn't restart after RocketChat server upgrade

### ISSUE THREAD

**Issue by anadahz** [42wim/matterbridge#1011]
**Describe the bug**
RocketChat matterbridge doesn't restart after RocketChat server upgrade.

**To Reproduce**

1. Upgrade RocketChat server
2. Bridge will not be able to connect
3. Bridge stalls

**Expected behavior**

RocketChat Bridge tries to reconnect to RocketChat server

**Screenshots/debug logs**

```
Dial error websocket.Dial wss://rocketchat.server:443/websocket: bad status
Subscription returned a nosub error map[id:3 msg:nosub] time="[Redacted]" level=error msg="SendMessage failed: {"details":{"method":"sendMessage"},"error":"error-invalid-user","errorType":"Meteor.Error","isClientSafe":true,"message":"Invalid user [error-invalid-user]","reason":"Invalid user"}" prefix=gateway
```

**Environment**
 - OS: Debian 10
 - Docker image: 42wim/matterbridge:1.16.3
 - Matterbridge version: version 1.16.3 84bfa8a

**Comment by anadahz** [comment]
Perhaps related to https://github.com/42wim/matterbridge/issues/855 .

**Comment by 42wim** [comment]
Yes, same issue, closed in favor of #855
